__docs__/wonder-blocks-form/labeled-text-field.stories.tsx

```diff
@@ -528,11 +520,8 @@ ErrorLight.args = {
 };
 
 ErrorLight.parameters = {
-    docs: {
-        description: {
-            story: `If an input value fails validation and the
-        \`light\` prop is true, \`TextField\` will have light error styling.`,
-        },
+    backgrounds: {
```

Comment:
We can use the Storybook backgrounds parameter for setting the background color so we don't have to provide wrappers around the components!